aiida/backends/tests/cmdline/commands/test_status.py

```diff
@@ -29,11 +29,13 @@ def test_status_1(self):
         """Test running verdi status.
 
         Note: The exit status may differ depending on the environment in which the tests are run.
+            Also cannot check for the exit status to see if connecting to all services worked, because
+            the profile might not be properly setup in this temporary config instance unittest.
         """
         options = []
         result = self.cli_runner.invoke(cmd_status.verdi_status, options)
         self.assertIsInstance(result.exception, SystemExit)
-        self.assertIn("profile", result.output)
-        self.assertIn("postgres", result.output)
-        self.assertIn("rabbitmq", result.output)
-        self.assertIn("daemon", result.output)
+        self.assertIn('profile', result.output)
+        self.assertIn('postgres', result.output)
```

Comment:
in order to prevent regression of this bug, perhaps we should search for `Connected to` as well?

Reply:
hm, but would not help on travis

Reply:
scratch that